### PR TITLE
Small fixes in proxy repositories documentation

### PR DESCRIPTION
**Pull request by rbonvall**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @rbonvall,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: src/reference/02-DetailTopics/03-Dependency-Management/04-Proxy-Repositories.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ This repositories file is all that's required to use a proxy repository.  These
 
 In case you need to define credentials to connect to your proxy repository, define an environment variable `SBT_CREDENTIALS` that points to the file containing your credentials:
 
-    export SBT_CREDENTIALS="\$HOME/.ivy2/.credentials"
+    export SBT_CREDENTIALS="$HOME/.ivy2/.credentials"
```
Comment (dwijnand) [inline src/reference/02-DetailTopics/03-Dependency-Management/04-Proxy-Repositories.md]: I think this one is right, it's to escape pandoc

Comment (rbonvall) [reply src/reference/02-DetailTopics/03-Dependency-Management/04-Proxy-Repositories.md]: Hi Dale, in a code block there is no need to escape special characters. Before my changes, the block was being parsed as a regular paragraph.

```
$ cat test.md
---
title: Test
---
Regular paragraph

  abc $xyz$
  abc \$xyz$

Code block:

    abc $xyz$
    abc \$xyz$

$ pandoc test.md -t html
<p>Regular paragraph</p>
<p>abc <span class="math inline"><em>x</em><em>y</em><em>z</em></span> abc $xyz$</p>
<p>Code block:</p>
<pre><code>abc $xyz$
abc \$xyz$</code></pre>
```

**Comment by rbonvall** [comment]
@lightbend-cla-validator I have already signed the CLA.

File: src/reference/02-DetailTopics/03-Dependency-Management/04-Proxy-Repositories.md (hunk 1 above)
Comment (dwijnand) [reply src/reference/02-DetailTopics/03-Dependency-Management/04-Proxy-Repositories.md]: ah ok, thanks

**Review by dwijnand — APPROVED** [review]
thanks!

**Pull request merged**